### PR TITLE
[DMS-228] Refinement code coverage

### PR DESCRIPTION
**Pull request by msilesgap**
- Exclude projects from the code coverage report
- Add documention about code coverage and how to generate the report
- Set the Threshold value to 55 since the minimun value is 58

### CODE REVIEW

File: docs/RUNNING-LOCALLY.md
Comment (simpat-adam) [inline docs/RUNNING-LOCALLY.md:1]: Avoid long lines in markdown with this extension: https://marketplace.visualstudio.com/items?itemName=stkb.rewrap. The command to wrap the whole file is `ctrl+A` `ctrl+Q`

**Review by simpat-adam — CHANGES_REQUESTED** [review]

File: docs/RUNNING-LOCALLY.md
Comment (msilesgap) [reply docs/RUNNING-LOCALLY.md:1]: Added, for mac we have to use `option+Q`

File: docs/RUNNING-LOCALLY.md [hunk 1]
```diff
@@ -37,3 +37,46 @@ cd ../
 > development environment testing. This file should be based on the
 > `appsettings.json` file on EdFi.DataManagementService.Api.Tests.Integration
 > folder.
+## Running Unit Tests and Generate Code Coverage Report
+To run the unit tests locally, execute the following command in a PowerShell
```
Comment (stephenfuqua) [inline docs/RUNNING-LOCALLY.md:41]: Please add an extra line above and below the header (line 40). Headers should always have space around them.

File: docs/RUNNING-LOCALLY.md [hunk 2]
```diff
@@ -37,3 +37,46 @@ cd ../
...
+terminal:
+
+```shell
+# From base directory
+./build.ps1 UnitTest -Configuration Debug
+```
+
+The previous command should generate two files in the base directory, which
+contain all the merged results from the unit tests execution.
+```
```
Comment (stephenfuqua) [inline docs/RUNNING-LOCALLY.md:51]: Code blocks are like headers in that they too are better with space around them.

File: docs/RUNNING-LOCALLY.md [hunk 3]
```diff
@@ -37,3 +37,46 @@ cd ../
...
+results.
+
+### Coverlet Parameters
+We are currently evaluating `line` and `branch` metrics from the Total
+coverage. If the total coverage is less than our threshold, the build will fail.
+
+Total: Ensures the total combined coverage result of all modules isn't less than
+the threshold
+
+```
```
Comment (stephenfuqua) [inline docs/RUNNING-LOCALLY.md:75]: Suggest adding "none" as the language for the code block.

File: docs/RUNNING-LOCALLY.md (hunk 1 above)
Comment (msilesgap) [reply docs/RUNNING-LOCALLY.md:41]: Fixed

File: docs/RUNNING-LOCALLY.md (hunk 2 above)
Comment (msilesgap) [reply docs/RUNNING-LOCALLY.md:51]: Added

**Review by simpat-adam — APPROVED** [review]

**Pull request merged**